native/android/fastlane/README.md

```diff
@@ -1,5 +1,5 @@
-fastlane documentation
-================
+# fastlane documentation
```

Comment:
I think these fastlane readmes are autogenerated and should maybe be excluded.

Reply:
done

Reply:
done